src/webgl/p5.RendererGL.js

```diff
@@ -620,17 +616,29 @@ p5.RendererGL.prototype.loadPixels = function() {
     return;
   }
   var pd = this._pInst._pixelDensity;
-  var x = arguments[0] || 0;
-  var y = arguments[1] || 0;
-  var w = arguments[2] || this.width;
-  var h = arguments[3] || this.height;
+  var x = 0;
+  var y = 0;
+  var w = this.width;
+  var h = this.height;
   w *= pd;
   h *= pd;
-  var pixels = new Uint8Array(
-    this.GL.drawingBufferWidth * this.GL.drawingBufferHeight * 4
+  //if there isn't a renderer-level temporary pixels buffer
+  //make a new one
+  if (this.pixels === undefined) {
```

Comment:
can you make this
`if (typeof this.pixels === 'undefined') {`
this is nit-picking, but we've generally stuck to checking typeof across the library. this post (second answer) is old and isn't particularly convincing, but since we've been going with this anyway, may as well stick with it?
https://stackoverflow.com/questions/4725603/variable-undefined-vs-typeof-variable-undefined

Reply:
Oh! I knew this at one point but probably forgot along the way. Thank you for catching this and reminding me of the convention!